.changeset/perfect-parrots-appear.md

```diff
@@ -0,0 +1,8 @@
+---
+"@atproto/ozone": patch
+"@atproto/bsky": patch
+"@atproto/api": patch
+"@atproto/pds": patch
+---
+
+Adds `postgate` records to power quote gating and detached quote posts, plus `hiddenReplies` to the `threadgate` record.
```

Comment:
I think we can remove ozone from here since the change is incidental to ozone (e.g. doesn't need this line in its changelog).  It'll still get bumped by virtue of the update to its `@atproto/api` version.
```suggestion
---
"@atproto/bsky": patch
"@atproto/api": patch
"@atproto/pds": patch
---

Adds `postgate` records to power quote gating and detached quote posts, plus `hiddenReplies` to the `threadgate` record.
```